test/active_job/queue_adapters/solid_queue_test.rb

```diff
@@ -3,6 +3,8 @@
 class ActiveJob::QueueAdapters::SolidQueueTest < ActiveSupport::TestCase
   include ActiveJob::QueueAdapters::AdapterTesting
 
+  include DispatchJobs
+
```

Comment:
nit: remove this empty line:
```suggestion
  include ActiveJob::QueueAdapters::AdapterTesting
  include DispatchJobs

```